cmd/capibmadm/cmd/vpc/key/create.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package key
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+
+	logf "sigs.k8s.io/cluster-api/cmd/clusterctl/log"
+
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/clients/iam"
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/clients/vpc"
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/options"
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/utils"
+)
+
+type keyCreateOptions struct {
+	name              string
+	publicKey         string
+	resourceGroupName string
+}
+
+// CreateCommand vpc key create command.
+func CreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create VPC key",
+		Example: `
+# Create key in VPC
+export IBMCLOUD_API_KEY=<api-key>
+capibmadm vpc key create --name <key-name> --region <region> --public-key "<public-key-string>"
+Using file-path to VPC key : capibmadm vpc key create --name <key-name> --region <region> --key-path <path/to/vpc/key>
+`,
+	}
+
+	options.AddCommonFlags(cmd)
+	var keyCreateOption keyCreateOptions
+	var filePath string
+	cmd.Flags().StringVar(&keyCreateOption.name, "name", keyCreateOption.name, "Key Name")
+	cmd.Flags().StringVar(&filePath, "key-path", "", "The absolute path to the VPC key file.")
```

Comment:
`VPC key` file sounds conflicting, it would be better to address it as `SSH key` wherever applicable

Reply:
`VPC key` changed to `SSH key` wherever necessary.